scripts/ci-workflows.test.mjs

```
@@ -769,3 +776,18 @@ test('both native platforms upload Maestro artifacts unconditionally', () => {
 		assert.equal(upload.if, 'always()', `${job} collects Maestro artifacts conditionally`);
 	}
 });
+
+test('Android clean-start flows dismiss a queued system ANR before waiting for Expo', () => {
+	for (const filename of ['01-clean-launch-connect.yml', '02-auth-setup.yml']) {
+		const flow = readMaestroFlow(filename);
+		const androidLaunch = flow.find(
+			(command) => command.runFlow?.when?.platform === 'Android'
```

ESLint

[error] Replace `⏎↹↹↹(command)·=>·command.runFlow?.when?.platform·===·'Android'` with `(command)·=>·command.runFlow?.when?.platform·===·'Android')`

(prettier/prettier)